### PR TITLE
e2e Tests for LNBits and BlueWallet

### PR DESCRIPTION
**Pull request by escapedcat**
re-adding commented out tests

### CODE REVIEW

File: tests/e2e/createWallets.spec.ts [hunk 1]
```diff
@@ -66,103 +66,107 @@ test.describe("Create or connect wallets", () => {
...
+      getByText($document, "Do you have a lightning wallet?")
+    );
 
-  //   // click at "LNDHub (BlueWallet)"
-  //   const createNewWalletButton = await getByText(
-  //     $document,
-  //     "LNDHub (Bluewallet)"
-  //   );
-  //   createNewWalletButton.click();
+    // click at "LNDHub (BlueWallet)"
```
Comment (kiwiidb) [inline tests/e2e/createWallets.spec.ts:150]: What if Bluewallet is down again and this causes the tests to fail?

Comment (escapedcat) [reply tests/e2e/createWallets.spec.ts:150]: Then we could create an issue and need to skip this test till it's fixed.

**Comment by bumi** [comment]
Does it make sense to put the first `Protect your wallet` steps into a helper function? this looks repetitive.

**Comment by escapedcat** [comment]
> Does it make sense to put the first `Protect your wallet` steps into a helper function? this looks repetitive.

Good point. Added them into the same file for now:
- `commonCreateWalletUserCreate`
- `commonCreateWalletSuccessCheck`

**Review by bumi — APPROVED** [review]

**Pull request merged**